apps/teams-test-app/e2e-test-data/barCode.json

```diff
@@ -42,6 +42,7 @@
     {
       "title": "scanBarCode API Call - Failure",
       "type": "callResponse",
+      "version": ">2.16.0",
```

Comment:
Just curious, from the current PR changes, and since only three APIs have the version: will other APIs need the version field?